### PR TITLE
572 Back navigation

### LINKED ISSUE #572: Back button to actually go one step back, instead of resetting to route view

### PR DESCRIPTION
**Pull request by kronken**
Closes #572

### CODE REVIEW

**Comment by Divinare** [comment]
Testing:

Bug when removing selected route:
1. At http://localhost:3000/routes/?routes[0]=0033&routes[1]=0812
1. Try to deselect route 0812
--> nothing happens. Second click takes to home

1. At http://localhost:3000/routes/?routes[0]=0033
1. Click close button
1. Route on map stays selected


Make card for this:

1. At http://localhost:3000/network/node/4160206/?routes[0]=0033
Console error/warnings

1) mobx.module.js:3083 [mobx.array] Attempt to read an array index (0) that is out of bounds (0). Please check length first. Out of bound indices will not be tracked by MobX

2) index.js:2178 Warning: `value` prop on `input` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.

Card: #583
At: http://localhost:3000/network/node/4160021/?routes[0]=0033
Scrollbar

File: src/components/sidebar/routesView/RouteShow.tsx [hunk 1]
```diff
@@ -33,36 +32,23 @@ class RouteShow extends React.Component<IRouteShowProps> {
...
     }
 
-    private closeRoute = () => {
-        // TODO: Move actual logic somwhere else, so this function only navigates to new url
-        this.props.routeStore!.removeFromRoutes(this.props.route.id);
-        const closeRouteLink = routeBuilder
-            .to(subSites.current)
-            .remove(QueryParams.routes, this.props.route.id)
-            .toLink();
-        navigator.goTo(closeRouteLink);
```
Comment (Divinare) [inline src/components/sidebar/routesView/RouteShow.tsx]: This code should be brought back

Comment (kronken) [reply src/components/sidebar/routesView/RouteShow.tsx]: Correct, this is now fixed

**Comment by Divinare** [comment]
Review ok

**Pull request merged**